### PR TITLE
Add decoders for broker and stop-limit data

### PR DESCRIPTION
**Pull request by mzhu25**
## Description
Adds decoding equivalents to the encoders found in `chainlink_utils` and `gods_unchained_utils`

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: contracts/broker/src/gods_unchained_utils.ts [hunk 1]
```diff
@@ -1,42 +1,59 @@
...
-            { name: 'proto', type: 'uint16' },
-            { name: 'quality', type: 'uint8' },
-        ]).encode({ proto, quality });
-        const data = dataEncoder.encode({ godsUnchainedAddress, validatorAddress, propertyData });
-        return assetDataUtils.encodeERC1155AssetData(brokerAddress, [], [new BigNumber(bundleSize)], data);
-    },
-};
+export interface GodsUnchainedProperties {
+    proto: BigNumber;
+    quality: BigNumber;
```
Comment (dorothy-zbornak) [inline contracts/broker/src/gods_unchained_utils.ts]: would the abi encoder accept these if they were just `number`s?

File: contracts/integrations/test/broker/broker_test.ts [hunk 2]
```diff
@@ -718,4 +712,30 @@ blockchainTests.resets('Broker <> Gods Unchained integration tests', env => {
             balanceStore.assertEquals(expectedBalances);
         });
     });
+
+    describe('Data encoding/decoding tools', () => {
+        const MAX_UINT8 = new BigNumber(2).exponentiatedBy(8).minus(1);
+        const MAX_UINT16 = new BigNumber(2).exponentiatedBy(16).minus(1);
```
Comment (dorothy-zbornak) [inline contracts/integrations/test/broker/broker_test.ts]: Nit: Seems clearer to just do like `new BigNumber(2**8-1)`

File: contracts/integrations/test/stop-limit/chainlink_stop_limit_test.ts [hunk 3]
```diff
@@ -15,12 +27,13 @@ import { LocalBalanceStore } from '../framework/balances/local_balance_store';
 import { DeploymentManager } from '../framework/deployment_manager';
 import { ChainlinkStopLimitContract, TestChainlinkAggregatorContract } from '../wrappers';
 
-blockchainTests.resets('Chainlink stop-limit order tests', env => {
+blockchainTests.resets.only('Chainlink stop-limit order tests', env => {
```
Comment (dorothy-zbornak) [inline contracts/integrations/test/stop-limit/chainlink_stop_limit_test.ts]:
```suggestion
blockchainTests.resets('Chainlink stop-limit order tests', env => {
```

File: contracts/broker/src/gods_unchained_utils.ts (hunk 1 above)
Comment (mzhu25) [reply contracts/broker/src/gods_unchained_utils.ts]: ah yeah, changed to `BigNumber | number`

**Pull request merged**